Purchases/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -153,25 +153,23 @@ class SubscriberAttributesManager {
 
     func handleAttributesSynced(syncingAppUserId: String, currentAppUserId: String, error: Error?) {
         if error == nil {
-            Logger.rcSuccess(String(format: Strings.attribution.attributes_sync_success, syncingAppUserId))
+            Logger.rcSuccess(Strings.attribution.attributes_sync_success(appUserID: syncingAppUserId))
             if syncingAppUserId != currentAppUserId {
                 deviceCache.deleteAttributesIfSynced(appUserID: syncingAppUserId)
             }
         } else {
             let receivedNSError = error as NSError?
-            Logger.error(String(format: Strings.attribution.attributes_sync_error,
-                                receivedNSError?.localizedDescription ?? "",
-                                receivedNSError?.userInfo ?? ""))
+            Logger.error(Strings.attribution.attributes_sync_error(details: receivedNSError?.localizedDescription,
```

Comment:
it seems like we could update the api for `attributes_sync_error` to accept an optional `NSError` instead of the `detail` and `userInfo`

Reply:
good call, I'll update